src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/WebSocketResources.cs

```diff
@@ -99,7 +100,7 @@ internal class WebSocketHttpsDuplexBinaryStreamedResource : EndpointResource<WSD
 
         protected override int GetPort(ResourceRequestContext context)
         {
-            return context.BridgeConfiguration.BridgeWebSocketPort + 1;
+            return context.BridgeConfiguration.BridgeWebSocketPort;
```

Comment:
Yes, the +1 actually screws things up when using certs based on how the cert installs the cert.